### DISCUSSION TITLE
Agenda for the Flatcar Office Hours on Tuesday, 11th of October 2022 17:30 CEST / 11:30am EDT / 8:30am PDT

### DISCUSSION THREAD

**Discussion by miao0miao** [flatcar/Flatcar#862] [Flatcar Office Hours]
## Agenda for the Flatcar Office Hours on Tuesday, 11th of October 2022  17:30 CEST / 11:30am EDT / 8:30am PDT

- Jitsi: https://meet.flatcar.org/OfficeHours
- Youtube live stream / recording: [link](https://www.youtube.com/watch?v=chY_Y-pqD2k)
- Community Calls calendar (all future calls): [Google Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [ical](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)

## Welcome
- Brief intro/check-in of the new participants in the Zoom call. Please introduce yourself and share what brings you here today.

## Agenda items
(Work in progress, please add your suggestions as comments!)
- New [Jitsi instance](https://github.com/flatcar/jitsi-server/) for our meetings (@t-lo )
- Summary of the Image-based Linux summit in Berlin (@bluca)
- Progress on ARM64 SDK (@jepio ?)
- SPDX SBOM (@jepio ?)
- Inclusion of SLSA provenance in Alpha and Beta (@t-lo ) - see https://www.flatcar.org/docs/latest/reference/supply-chain/
- New pipeline now in production (@t-lo ) - see https://github.com/flatcar/scripts/tree/main/ci-automation
- Flatcar on ARM64 Azure Marketplace (@sayanchowdhury) 
- [Kernel updates workflow](https://docs.google.com/document/d/1-Jd0cKTlkWYbl2WJNXtG2_0vpcJ5fqbbM9R1v5jW4Wg/edit?usp=sharing) (@krnowak )
- User / maintainer sync on #378 , data and telemetry to help debugging this, and next steps (@seh )

## Release planning
- Stable patch releases now 1 week later to let kernel soak in Beta for a week (@sayanchowdhury )
- Ongoing and upcoming releases (https://github.com/orgs/flatcar/projects/7/views/8)

## Q&A
- Questions from community participants, answered by the Flatcar maintainers

**Comment by krnowak** [comment]
I'd like to talk about kernel updates workflow. Currently writing the doc about it: https://docs.google.com/document/d/1-Jd0cKTlkWYbl2WJNXtG2_0vpcJ5fqbbM9R1v5jW4Wg/edit?usp=sharing

**Reply by krnowak** [reply]
Comments in the docs are welcome, of course.

**Comment by t-lo** [comment]
Some thoughts / topics we could touch:
- New Jitsi instance for our meetings (@t-lo )
- Summary of the Image-based Linux summit in Berlin (@bluca)
- ARM64 SDK (@jepio ?)
- SPDX SBOM (@jepio ?)
- New pipeline now in production (@t-lo )
- Inclusion of SLSA provenance in Alpha and Beta (@t-lo )
- Stable patch releases now 1 week later to let kernel soak in Beta for a week (@sayanchowdhury ?)
- Flatcar on ARM64 Azure Marketplace (@sayanchowdhury ?) 
- User / maintainer sync on #378 , data and telemetry to help debugging this, and next steps (@seh )

**Reply by seh** [reply]
Do you ever discuss outstanding GitHub issues during these calls? #378 remains a major problem for us, and I don't know if it's actually a problem with Flatcar, or just with the Linux kernel, regardless of distribution. I'll attend the next meeting if it's a fitting forum for such topics.

**Reply by jepio** [reply]
@seh we discuss any topic someone brings to the community call. #378 is tricky because we have no way of reproducing, and we use a fairly vanilla upstream kernel so it’s likely an upstream issue. In cases like this we need more help from the reporter with debugging and reporting to LKML.

**Reply by t-lo** [reply]
I've added an agenda item to discuss debugging info that could help us (and how to get the info), and to brainstorm on next steps.

**Reply by t-lo** [reply]
And yes, to answer the original question, the office hours are a good place to bring issues to attention. Sometimes day-long back-and-forth on an issue can be cut to a few minutes of video chat :)

**Comment by pothos** [comment]
Updated the link to https://github.com/orgs/flatcar/projects/7/views/8
